src/Pagination/Pagination.xml

```diff
@@ -65,6 +66,36 @@
                     <category>General</category>
                     <description>Including the ... buttons, minimal value 7, recommend an uneven number</description>
                 </property>
+                <property key="renderPageSizeAs" type="enumeration" defaultValue="dropdown">
+                    <caption>Render page size</caption>
+                    <category>General</category>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="dropdown">Drop down</enumerationValue>
+                        <enumerationValue key="input">Input</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </properties>
+        </property>
+        <property key="pageSizeOptions" type="object" isList="true" required="false">
+            <caption>Options</caption>
+            <category>Page size</category>
+            <description>Dropdown options for 'Paging style' 'Custom'. The options are used the for the item 'Page size' when 'Render page size' is 'Drop down'</description>
```

Comment:
.... The options are `used for the item` ...

Reply:
Thanks

Reply:
Done